nemo_curator/modules/config.py

```diff
@@ -80,6 +81,9 @@ class FuzzyDuplicatesConfig(BaseConfig):
     bucket_mapping_blocksize: int = 256
     parts_per_worker: int = 1
     bucket_parts_per_worker: int = 8
+    # String bytes limit for cuDF
+    # https://github.com/rapidsai/cudf/issues/13733
+    max_text_bytes_per_part: int = int(np.iinfo(np.int32).max * 3)
```

Comment:
I want us to not use this config at all. 

@ayushdg , @praateekmahajan , @sarahyurick  lets huddle before we merge this in